challenges/chat_app_css_illustration/tailwind.config.js

```diff
@@ -26,7 +26,7 @@ module.exports = {
       },
       screens: {
         mobile: '375px',
-        desktop: '1440px',
+        desktop: '1200px',
       },
```

Comment:
Same as in the other PR, use default Tailwind breakpoints.

Reply:
Changed to default Tailwind breakpoints